core/arch/aarch64/instr_create.h

```diff
@@ -38,12 +38,17 @@
 #include "../instr_create_shared.h"
 #include "instr.h"
 
+/* DR_API EXPORT TOFILE dr_ir_macros_aarch64.h */
+/* DR_API EXPORT BEGIN */
+
+/** Denotes half-precision floating point vector elements. */
```

Comment:
I would suggest briefly including context: "Used in an additional immediate source operand to a vector operation, denotes ...." or something.

Reply:
Maybe put a link to the bt.dox section via `\ref sec_IR_AArch64`

Reply:
Done, thanks!